spec/runners/active_encode_derivatives_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 
 describe Hydra::Derivatives::ActiveEncodeDerivatives do
-  context '.create' do
+  context 'when create' do
```

Comment:
I think this should change to `describe '.create' do` instead.

Reply:
done